### PR TITLE
Referi merge

### LINKED ISSUE #57: no info about geometry of field in matlab

### LINKED ISSUE #58: no way to control voltage of kicker from algo

### LINKED ISSUE #104: Add necessary dll to result archive on AppVeyor
Была ругань на libspp-0.dll у @DmiitriyJarosh.
Хорошо бы протестировать на совсем чистой винде.

### PR DESCRIPTION
**Pull request by DmiitriyJarosh**
All accessible fields of robot control added to matlab, referi merged into branch and small fixes were done (remote control for simultor, axis orientation) Also data about ball sensor added to every robot for matlab
Close #119
Close #75
Close #120
Close #57
Close #58
Close #104

### CODE REVIEW

**Comment by DmiitriyJarosh** [comment]
Close #120

**Comment by DmiitriyJarosh** [comment]
Close #119 
Close #75

**Comment by anastasiia-kornilova** [comment]
@DmiitriyJarosh, add comment to pull-request description, please.

**Comment by DmiitriyJarosh** [comment]
Solve #57 #58 #104

File: LARCmaCS/connector.cpp [hunk 1]
```diff
@@ -53,46 +53,36 @@ void Connector::onConnectorChange(bool isSim, const QString &ip, int port)
...
-			if (!mIsPause) {
-				DefaultRobot::formControlPacket(command, k, rule[k].mSpeedX, rule[k].mSpeedY, rule[k].mSpeedR,
-						rule[k].mKickUp, rule[k].mKickForward, rule[k].mKickerVoltageLevel,
-												rule[k].mDribblerEnable, rule[k].mSpeedDribbler);
+	if (!mIsPause) {
+		for (int k = 0; k < rule.size(); k++) {
+			QByteArray command;
+			bool simFlag = mIsSim;
+			if (!simFlag) {
+				if (!mIsPause) {
```
Comment (anastasiia-kornilova) [inline LARCmaCS/connector.cpp]: Unneseccary if

File: LARCmaCS/connector.cpp [hunk 2]
```diff
@@ -53,46 +53,36 @@ void Connector::onConnectorChange(bool isSim, const QString &ip, int port)
...
+					DefaultRobot::formControlPacket(command, k, rule[k].mSpeedX, rule[k].mSpeedY, rule[k].mSpeedR,
```
Comment (anastasiia-kornilova) [inline LARCmaCS/connector.cpp]: Please, add similar shift from the begging for all lines below

File: LARCmaCS/connector.cpp [hunk 3]
```diff
@@ -53,46 +53,36 @@ void Connector::onConnectorChange(bool isSim, const QString &ip, int port)
...
+							rule[k].mKickUp, rule[k].mKickForward, rule[k].mKickerVoltageLevel,
+													rule[k].mDribblerEnable, rule[k].mSpeedDribbler, rule[k].mAutoKick,
+													rule[k].mKickerChargeEnable, rule[k].mBeep);
+				} else {
+					DefaultRobot::formControlPacket(command, k, 0, 0, 0, 0, 0, 0, 0);
+				}
 			} else {
-				DefaultRobot::formControlPacket(command, k, 0, 0, 0, 0, 0, 0, 0);
+				if (!mIsPause) {
```
Comment (anastasiia-kornilova) [inline LARCmaCS/connector.cpp]: mPause state was checked outside

File: LARCmaCS/connector.h [hunk 4]
```diff
@@ -27,11 +27,10 @@ public slots:
 	void onConnectorChange(bool isSim, const QString & ip, int port);
 	void sendNewCommand(const QVector<Rule> & rule);
 	void onPauseChanged(bool status);
-
-private:
```
Comment (anastasiia-kornilova) [inline LARCmaCS/connector.h]: Why? All logic about sender should be hidden in connector.

File: LARCmaCS/larcmacs.cpp [hunk 5]
```diff
@@ -54,6 +55,8 @@ LARCmaCS::LARCmaCS(QWidget *parent)
 	//remotecontrol
 	connect(&remotecontol, SIGNAL(RC_control(int, int, int, int, bool)),
 			this, SLOT(remcontrolsender(int, int, int, int, bool)));
+	connect(this, SIGNAL(run(int, const QByteArray &)), &connector, SLOT(run(int, const QByteArray &)));
```
Comment (anastasiia-kornilova) [inline LARCmaCS/larcmacs.cpp]: It is better to use common interface for connector, don't specify here the type of robot communication

**Pull request merged**